### PR TITLE
Blocks-77 - Render formulas like in Catroid

### PR DESCRIPTION
**Pull request by whati001**
Render formulas like in Catroid. Updated catroid string files. Use 'en_GB' language as fallback for absence values in other langs. Changed default values of bricks.
https://jira.catrob.at/projects/BLOCKS/issues/BLOCKS-77

### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/js/blocks/categories/arduino.js [hunk 1]
```diff
@@ -11,12 +11,12 @@ export default {
       {
         "type": "field_input",
         "name": "ARDUINO_DIGITAL_PIN_NUMBER",
-        "text": "DEFAULT_VALUE"
+        "text": "unset"
```
Comment (whati001) [inline src/js/blocks/categories/arduino.js:14]: Replaced default value for each brick.

File: src/js/parser/formula.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    })();
+    const right = (() => {
+      if (f.right)
+        return Formula.stringify(f.right);
+      return '';
+    })();
+
+    const nodeValue = Formula.packLayout(f.operator, f.value, left, right);
+    return nodeValue;
+  }
```
Comment (whati001) [inline src/js/parser/formula.js:102]: Stringify method, refactored concatFormula.

**Comment by whati001** [comment]
The huge amount of change files is because of updated the catroid string and share test files. Also because of changing the default value from the bricks

**Comment by Hiebl** [comment]
Wolfgang Slany: "So, this is not yet using the Crowdin data, right? BTW, actually for English it is sine and cosine. Sinus and cosinus is the German version. Why does it work for the others and not for these as well as sqrt etc?"

**Comment by Hiebl** [comment]
Wolfgang Slany: "Also, one thing that's still missing is the differentiation between text strings, e.g., 'hello world', versus names of functions such as pi, versus variable names such as "my variable", and list names such as * my list * (without the spaces). In the future, once the user defined bricks are finished (soon), there will also be the parameters [my x] (like variables, but cannot be changed and only used in their respective definition script."

**Pull request closed without merging**